ArduPlane/tailsitter.cpp

```diff
@@ -277,9 +277,9 @@ bool QuadPlane::tailsitter_transition_vtol_complete(void) const
             return true;
         }
     }
-    if (labs(plane.ahrs.pitch_sensor) > tailsitter.transition_angle*100 ||
-        labs(plane.ahrs.roll_sensor) > tailsitter.transition_angle*100 ||
-        AP_HAL::millis() - transition_start_ms > 2000) {
+    if (labs(plane.ahrs.pitch_sensor) > get_tailsitter_transition_angle_vtol()*100 ||
+        labs(plane.ahrs.roll_sensor) >  get_tailsitter_transition_angle_vtol()*100 ||
+        AP_HAL::millis() - transition_start_ms > uint32_t(MIN (4000, tailsitter.transition_time_vtol_ms))) { // but dont wait longer than 4 seconds to complete transition
```

Comment:
I actually don't think the 4 seconds min is required, maybe just a hard coded 1.5 or 2x

```suggestion
        AP_HAL::millis() - transition_start_ms >  tailsitter.transition_time_vtol_ms.get()*2 ) {
```

Reply:
its 4 sec max....not min....figured you may want a slow pitch up but not for it go on for a long time (rates would be handy here...could set low rate, but shorter time out)....guess making just the time would do, since the ramp is based on 1/2 the time

Reply:
changed to compare to time directly , like it is in forward transition